src/modules/CRUD/components/Details/Edit/index.js

```diff
@@ -17,7 +19,7 @@ export default withRouter(
     };
   })(
     connectCRUDProvider(({
-      layersList,
+      CRUDSettings,
```

Comment:
Why `CRUDSetting`, capitalised, like a class. We are in CRUD context, so a single `settings` should be enough to know this is the CRUD'settings.